package.json

```diff
@@ -2,8 +2,11 @@
   "name": "root",
   "private": true,
   "scripts": {
+    "prettier:check": "npm run prettier -- --check",
```

Comment:
that's quite cool, i totally forgot that you can pass args to another `npm run` command! I'd just move the `:check` versions after the normal commands

Reply:
Yes, the previous solution was getting too much verbose. I made an amend to move the checks.